src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -65,6 +79,15 @@ const NavHeader = (props) => {
     [setDomainHeight]
   );
 
+  const onSearchLayout = React.useCallback(
+    (event) => {
+      const {height} = event.nativeEvent.layout;
+      setSearchHeight(Math.round(height));
+      getSearchLayout(Math.round(height));
+    },
+    [setSearchHeight]
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onSearchLayout` function is well defined using `React.useCallback`, which will prevent unnecessary re-renders. Good job!



<!-- This is an auto-generated comment by OSS CodeRabbit -->